src/overlays/actors/ovl_En_Insect/z_en_insect.c

```diff
@@ -10,30 +10,39 @@
 
 #define FLAGS 0
 
+#define IS_DROPPED(type) ((type) == INSECT_TYPE_FIRST_DROPPED || (type) == INSECT_TYPE_EXTRA_DROPPED)
+
```

Comment:
I dont see this macro as being too beneficial personally, I think id rather the type checks be inline as is